src/App.tsx

```diff
@@ -16,36 +17,82 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortedState, setSortedState] = useState<number>(0);
+  const [reverseList, setReverseList] = useState<boolean>(false);
+  const [changedGoods, setChangedGoods] = useState<string[]>(goodsFromServer);
+
+  const handleAlphaSort = (): void => {
+    if (sortedState !== 1) {
+      setSortedState(1);
```

Comment:
it's not a good idea to use numbers to reference multiple types of sorting or things that are not related to numbers such as Strings

that's because if someone were to read your code they wouldn't understand which state relates to which sort method without further reading the entire code